content/team/index.md

```diff
@@ -0,0 +1,258 @@
+---
+pagetype: major
+pagename: team
+section: Team
+title: Team | IPFS
+url: team
+save_as: team/index.html
+constellation: constellation-02.svg
+---
+
+The IPFS Team is a collection of thousands of contributors from multiple geographies and backgrounds, interested in making the Web Distributed.
+
+```
+TODO Insert nice animation with World Map and hex faces
+```
+
+To Work together collaboratively, distributed and asynchronously, [we've developed multiple habits for Synchronous and Asynchronous communication and teams to focus on different areas of focus](https://github.com/ipfs/team-mgmt#how-we-work-together). In this page, you can learn about the multiple IPFS Working Groups and how to join the conversation.
+
+## IPFS Working Groups
+
+The IPFS Working Groups are teams of people that are appointed to research, develop, and deploy work under the working groups' scope. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+You can learn more about this group structure at [TEAM_ROLES_STRUCTURES.md](https://github.com/ipfs/team-mgmt/blob/master/TEAMS_ROLES_STRUCTURES.md#working-groups) in the ipfs/team-mgmt repo.
+
+The active IPFS Working Groups list:
+
+- [**Project**](#project)
+- [**Community**](#community)
+- Development and Maintenance of the Protocol
+  - [**JS Core**](#js-core)
+  - [**Go Core**](#go-core)
+  - [**IPFS GUI**](#ipfs-gui)
+  - [**IPFS Cluster**](#ipfs-cluster)
+  - [**IPFS Infrastructure**](#ipfs-infrastructure)
+- Efforts to Support Specific Uses
+  - [**Integration with Web Browsers**](#integration-with-web-browsers)
+  - [**Dynamic Data and Capabilities**](#dynamic-data-and-capabilities)
+  - [**Decentralized Data Stewardship**](#decentralized-data-stewardship)
+  - [**Local Offline Collaboration**](#local-offline-collaboration-locol)
```

Comment:
The Markdown/CSS hates indented bullet points

<img width="620" alt="image" src="https://user-images.githubusercontent.com/1211152/52701723-2d5c0900-2f7b-11e9-8276-0aa08d9c3aa9.png">

@olizilla is this intentional?

Reply:
@daviddias No that's a bug. If you write the copy you want to seem I'll make it look sensible.

Reply:
@olizilla the padding around the sub-list doesn't seem right though. Is that intentional, or we just haven't had time to make super-clean bullets? (which is always a bit of a bear to create in a reusable way)?